client/src/pages/importprospects/index.js

```diff
@@ -0,0 +1,335 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { makeStyles, withStyles } from "@material-ui/core/styles";
+import clsx from "clsx";
+import Stepper from "@material-ui/core/Stepper";
+import Step from "@material-ui/core/Step";
+import StepLabel from "@material-ui/core/StepLabel";
+import Check from "@material-ui/icons/Check";
+import SettingsIcon from "@material-ui/icons/Settings";
+import GroupAddIcon from "@material-ui/icons/GroupAdd";
+import VideoLabelIcon from "@material-ui/icons/VideoLabel";
+import StepConnector from "@material-ui/core/StepConnector";
+import Button from "@material-ui/core/Button";
+import { Redirect } from 'react-router-dom';
+import Typography from "@material-ui/core/Typography";
+import UploadCSV from "./steps/UploadCSV";
+import { Container } from "@material-ui/core";
+import axios from "axios";
+import useSnackBar from "common/useSnackbar";
+import LinearProgress from "@material-ui/core/LinearProgress";
+import MapCSV from "./steps/MapCSV";
+import withAuth from "common/withAuth";
+import AddProspects from "./steps/AddProspects";
+
+const useQontoStepIconStyles = makeStyles({
+  root: {
+    color: "#eaeaf0",
+    display: "flex",
+    height: 22,
+    alignItems: "center",
+  },
+  active: {
+    color: "#784af4",
+  },
+  circle: {
+    width: 8,
+    height: 8,
+    borderRadius: "50%",
+    backgroundColor: "currentColor",
+  },
+  completed: {
+    color: "#784af4",
+    zIndex: 1,
+    fontSize: 18,
+  },
+});
+
+function QontoStepIcon(props) {
```

Comment:
Can we put QontoStepIcon, Colorlibconnector, Colorlibstepicon on separate file?

Reply:
I also find these names hard to understand what they are. Also let's camelcase importprospects folder